### PR TITLE
Enhancement/81

### PR DESCRIPTION
**Pull request by mattheww95**
Addressed comments in enhancement request #81

### CODE REVIEW

**Comment by mattheww95** [comment]
Tests are passing locally but not in CI.

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Updated samtools/minimap2 container fixing CI issues and issues running the pipeline with Docker. See [PR 85](https://github.com/phac-nml/mikrokondo/pull/85)
 
+### `Added`
+
+- Altered name of stored `SpeciesTopHit` field in the irida-next.config, and added a field displaying the field name used addressing (Issue 81)[https://github.com/phac-nml/mikrokondo/issues/81]
```
Comment (apetkau) [inline CHANGELOG.md:18]: Can you also reference this PR (PR 90)

File: conf/irida_next.config [hunk 2]
```diff
@@ -141,6 +142,8 @@ iridanext {
                     "QualityAnalysis.length.value",
                     "QualityAnalysis.nr_contigs.qc_status",
                     "QualityAnalysis.nr_contigs.value",
+                    "QCSummary",
```
Comment (apetkau) [inline conf/irida_next.config:145]: You have QCSummary twice in here. Can you remove one.

File: subworkflows/local/determine_species.nf [hunk 3]
```diff
@@ -40,18 +41,26 @@ workflow DETERMINE_SPECIES {
...
         reports = reports.mix(parsed.mash_out.map{
             meta, report -> tuple(meta, params.top_hit_species, report)
         })
+
         top_hit = parsed.mash_out
         versions = versions.mix(MASH_SCREEN.out.versions)
         versions = versions.mix(parsed.versions)
     }
 
+    // Create a channle identifying pipelines output ID
```
Comment (apetkau) [inline subworkflows/local/determine_species.nf:58]: Change **channle** to **channel**

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. Just a few small comments.

File: subworkflows/local/determine_species.nf (hunk 3 above)
Comment (mattheww95) [reply subworkflows/local/determine_species.nf]: fixed in: https://github.com/phac-nml/mikrokondo/pull/90/commits/d60495d41e4628d0233659b88f0baa0d0d50a549

File: CHANGELOG.md (hunk 1 above)
Comment (mattheww95) [reply CHANGELOG.md]: fixed in: https://github.com/phac-nml/mikrokondo/pull/90/commits/d60495d41e4628d0233659b88f0baa0d0d50a549

**Review by apetkau — APPROVED** [review]
Looks good @mattheww95

**Pull request merged**